tests/integration/regenerate.sh

```diff
@@ -184,7 +182,6 @@ regenerate_without_llm() {
       MAX_ITERATIONS=$MAX_ITERATIONS \
       FORCE_APPLY_PROMPTS=true \
       DEFAULT_AGENT=$agent \
-      TEST_RUNTIME="$TEST_RUNTIME" \
```

Comment:
I think we should still keep this and make this configurable? Bc we may add different runtime in the future?

Reply:
I don't understand why we would keep this dead code now when we try to clean up stuff in other areas? 🤔

Reply:
I'm trying to implement a different RemoteRuntime in a different PR, and when that finishes I likely want to add it to the runtime tests and integration test. Keeping this TEST_RUNTIME argument will make it easier for us to extend the tests to different runtime types in the future

Reply:
Ahh, ok, got it. Will do the reverts then.

Reply:
Thanks for all the hassle 😃

Reply:
All good, done now. 😄